robyn/argument_parser.py

```diff
@@ -60,6 +66,7 @@ def __init__(self) -> None:
         self.create = args.create
         self.docs = args.docs
         self.open_browser = args.open_browser
+        self.version = args.version
```

Comment:
I don't think that we need this

Reply:
Just added this so that someone can easily check if they have installed Royn correctly. Also added [docs]([docs/getting-started.md](https://github.com/sparckles/robyn/pull/556/files#diff-31bcba2ccafa41d46fbbd6d1219f7f1e3b1fb3cad9faa8e4dc521bbb579dd7b3)) for it.

Reply:
I think they can just run the CLI. There's no need to add another param in constructor class imo